### ISSUE TITLE
Add github action for automatically updating project stars

### ISSUE THREAD

**Issue by presmihaylov** [IonicaBizau/made-in-bulgaria#13]
Currently, the github stars of projects are hardcoded and they quickly get outdated.

It is possible to setup a CI step, which automatically pulls the latest github stars information for every project & update the package.json file accordingly.

**Comment by IonicaBizau** [comment]
True! I have a script for that, but haven't set it up... Will update this issue once it is ready.
